addons/mail/models/mail_thread.py

```diff
@@ -937,7 +937,7 @@ def message_route(self, message, message_dict, model=None, thread_id=None, custo
                 is_a_reply = False
                 rcpt_tos_valid_localparts = [to for to in rcpt_tos_valid_localparts if to in other_model_aliases.mapped('alias_name')]
 
-        if is_a_reply:
+        if is_a_reply and reply_model:
```

Comment:
please avoid links in your commit message (just the task id is enough)

also the line are too long (where githuib truncate the title you have to go to the next line)

Reply:
> also the line are too long (where githuib truncate the title you have to go to the next line)

do you mean this line?`[FIX] mass_mailing: Fix an edge case when reply_model is Null in the database` ?

Reply:
all lines of the commit message (title include if possible)

Reply:
> where github truncate the title you have to go to the next line

70 :) 

```
>>> len("[FIX] mass_mailing: Fix an edge case when reply_model is Null in the …")
70
```